oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticSuggestQueryTest.java

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
There's a test class for suggest use cases - IndexSuggestionCommonTest which is extended by ElasticIndexSuggestionCommonTest and LuceneIndexSuggestionCommonTest. Maybe we can add new tests there only - this way it's easier to maintain - also we do similar tests for both lucene and elastic.

Reply:
@nit0906 I tried there first, but realized it's lack of explain query etc. facilities in ElasticAbstractQueryTest, made it needs lots of additional code for making it possible.